src/analysis/inner_product_space/dual.lean

```diff
@@ -5,6 +5,7 @@ Authors: Frédéric Dupuis
 -/
 import analysis.inner_product_space.projection
 import analysis.normed_space.dual
+import analysis.normed_space.star
```

Comment:
Sorry not to have checked this out myself, but is this new import really needed?

Reply:
It's needed for the `ring_hom_isometric.star_ring_aut` instance, which is needed for the operator norm for conjugate-linear maps to be defined.